config/karma.conf.js

```diff
@@ -60,8 +60,18 @@ module.exports = function (config) {
       type: 'in-memory'
     },
 
+    sonarQubeUnitReporter: {
+      sonarQubeVersion: '5.x',
+      outputFile: 'reports/ut_report.xml',
+      overrideTestDescription: true,
+      testPath: 'src/app',
+      testFilePattern: '.spec.ts',
+      useBrowserName: false
+    },
+
     remapCoverageReporter: {
       'text-summary': null,
+      lcovonly: './coverage/coverage.lcov',
```

Comment:
can you move this to process.env.SONAR_QUBE unless this is for something else

Reply:
Done